config/routes.rb

```diff
@@ -307,6 +307,9 @@
       get 'auto_complete_search'
     end
   end
+
+  get 'menu', :to => 'user_menus#menu'
```

Comment:
should we consider changing the UI to use this URL? /cc @sharvit @amirfefer

Reply:
nice to have, but it barely affects performance (does it make sense to render something before the menu  itself?)  AFAIK the menu is static, only if we have a need for a realtime change (something like runtime enable/disable plugin)  it would be beneficial

Reply:
IMO it is nice that the UI can use it now, it would allow us to render the app a bit faster but without the menu items.
TBH I don't see it happen soon as there are other "small" things that can bring more value to the user.

Reply:
It can be useful when one wants to experiment with a fully standalone webpack implementation that only calls the API.

Reply:
Also if we render the menu for every page load, it is slower when the user is not admin due to all permissions checks.

Reply:
We actually currently cache the menu rendering as part of the react_app mounting here:
https://github.com/theforeman/foreman/blob/develop/app/views/layouts/base.html.erb#L60-L62

Reply:
Also, the new endpoint gives a flat list of menu items and urls, without any structure as we have currently.